### ISSUE TITLE
No field to set some zwave parameters

### ISSUE THREAD

**Issue by jiquem** [home-assistant/frontend#14422]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

The field to set a numeric parameters is too small to enter something on my ZMNHCD1 zwave parameters

<img width="688" alt="Capture d’écran 2022-11-21 à 07 48 56" src="https://user-images.githubusercontent.com/1716813/202983563-f0ebc1da-37b3-41fa-839e-a4839262e7d1.png">

### Describe the behavior you expected

A field allowing to enter a 5 digit number

### Steps to reproduce the issue

1.Install a ZMNHCD
2. Go in the zwave parameters
3. Try to set a number in field 74

### What version of Home Assistant Core has the issue?

2022.11.3

### In which browser are you experiencing the issue with?

Safari 18614.2.9.1.12 / Brave v1.45.127

### Which operating system are you using to run this browser?

MacOS Ventura

**Comment by sarahmva** [comment]
I am facing the same issue. As a workaround, I can set the mdc_text_field width css attribute to a fixed value and it works. There are a few other CSS values that can be adjusted to make it work. Or just make the browser window small enough to force skinny mode. 

![image](https://user-images.githubusercontent.com/68161487/207116885-f3cf0f3b-41eb-4959-bb16-c7dd5de558da.png)


![image](https://user-images.githubusercontent.com/68161487/207115246-adde9d52-8971-48b2-831f-22fee3c34bd3.png)

**Comment by karwosts** [comment]
Played with this a little bit, couldn't find a "good" fix, but this change makes all the boxes at least visible/usable for me:

`src/panels/config/integrations/integration-panels/zwave_js/zwave_js-node-config.ts`
```
         :host(:not([narrow])) ha-settings-row ha-textfield {
-          width: 30%;
+          width: 100px;
           text-align: right;
         }
```

Would have like to figure out some way to make the 30% work, but I'm not enough of a CSS expert to figure out all these styles. Not sure if such a PR would be accepted or not. I could try if nothing else comes up, seems pretty broken as-is.

**Comment by karwosts** [comment]
Fixed by #14788
